ansible/vars/topo_t0-isolated-d2u510s2.yml

```diff
@@ -10759,6 +11272,7 @@ configuration:
   ARISTA01PT0:
     properties:
     - common
+    - tor
```

Comment:
Hi @w1nda , you add the role "tor" for the 2 service port VMs, but we don't yet have the t0-isolated-d2u510s2-tor.j2 template. Looks like we need to add one? Now these 2 VMs are not created in add_topo.